app/upgrades/v4/README.md

```diff
@@ -0,0 +1,13 @@
+# v4 upgrade
+
+This upgrade moves Sommelier to major version 4.
+
+## Summary of changes
+
+* Switch to use of the upgrade module
+* Add the cork module for receiving arbitrary logic calls
+* Add the cellarfees module to provide a module account for cellar fees to be bridged to
+* Support sending to specified module accounts over the bridge
+* Community spend governance proposal for sending funds over the bridge to an Ethereum address
+* Fix a bug affecting the capitalization of ERC20 addresses in denominations
+* Fix a bug incorrectly setting the timeouts of ContractCallTxs
```

Comment:
Do we document the removal of allocation? Or does it not matter because it was never used?

Reply:
Good note, I'll add it to the README.